Epilepsy/epilepsy_utils.py

```diff
@@ -0,0 +1,62 @@
+import cPickle
+
+from collections import defaultdict
+from itertools import product
+from pandas import DataFrame
+from string import punctuation
```

Comment:
From https://www.python.org/dev/peps/pep-0008/#imports:

Imports should be grouped in the following order:

1. standard library imports
2. related third party imports
3. local application/library specific imports

You should put a blank line between each group of imports.